images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,133 @@
+}
+
+function init_send_network_topology {
+	case "$TOPOLOGY_UPDATE_INTERVAL" in
+		''|*[!0-9]*|0)
+			echo "Skipping topology cron: invalid TOPOLOGY_UPDATE_INTERVAL: $TOPOLOGY_UPDATE_INTERVAL" >&2
+			return 0
+			;;
+	esac
+
+	if [ -z "$TOPOLOGY_UUID" ]; then
+		export TOPOLOGY_UUID=$(get_redis_value "default_openvpn_topology_uuid")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)